cartesia/_async_websocket.py

```diff
@@ -100,12 +103,49 @@ async def no_more_inputs(self) -> None:
         await self.send(
             model_id=DEFAULT_MODEL_ID,
             transcript="",
-            output_format=TTS.get_output_format("raw_pcm_f32le_44100"),
+            output_format=TTS.get_output_format(DEFAULT_OUTPUT_FORMAT),
             voice_embedding=DEFAULT_VOICE_EMBEDDING,  # Default voice embedding since it's a required input for now.
             context_id=self._context_id,
             continue_=False,
         )
 
+    async def flush(self) -> Callable[[], AsyncGenerator[Dict[str, Any], None]]:
```

Comment:
in this case, if one flushes, they have to use the `generator` returned by `flush()`. they can't use the standard `receive` right?

Reply:
They technically can, it's just not advised